src/query/interpret/awesome_memgraph_functions.cpp

```diff
@@ -1325,6 +1350,7 @@ std::function<TypedValue(const TypedValue *, int64_t, const FunctionContext &ctx
   if (function_name == "PROPERTIES") return Properties;
   if (function_name == "RANDOMUUID") return RandomUuid;
   if (function_name == "SIZE") return Size;
+  if (function_name == "PROPERTY_SIZE") return PropertySize;
```

Comment:
underscore in the function name, not consistent with previous naming e.g. `RANDOMUUID`